examples/deployment/README.md

```diff
@@ -0,0 +1,731 @@
+# Coherence Operator Deployment Example
+
+This example showcases how to deploy Coherence applications using the Coherence Operator.
+
+The following scenarios are covered:
+
+1. Installing the Coherence Operator 
+1. Installing a Coherence cluster 
+1. Deploying a Proxy tier
+1. Deploying an storage-disabled application 
+1. Enabling Active Persistence
+
+After the initial install of the Coherence cluster, the following examples 
+build on the previous ones by issuing a `helm apply` to modify
+the install adding additional roles.
+
+You can use `helm create` for any of the examples to install that one directly.
+
+# Table of Contents
+
+* [Prerequisites](#prerequisites)
+  * [Coherence Operator Quick Start](#coherence-operator-quick-start) 
+  * [JDK and Maven Vvrsions](#jdk-and-maven-versions) 
+  * [Install Coherence into local Maven repository](#install-coherence-into-local-maven-repository)
+  * [Create the example namespace](#create-the-example-namespace)
+  * [Get Coherence Docker image](#get-coherence-docker-image)
+  * [Create a secret](#create-a-secret)
+  * [Clone the GitHub repository](#clone-the-github-repository)
+  * [Install the Coherence Operator](#install-the-coherence-operator)
+  * [Port Forward Grafana and Kibana ports](#port-forward-grafana-and-kibana-ports)
+* [Run the Examples](#run-the-examples)
+  * [Example 1 - Coherence cluster only](#example-1---coherence-cluster-only)
+  * [Example 2 - Adding a Proxy role ](#example-2---adding-a-proxy-role) 
+  * [Example 3 - Adding a User application role](#example-3---adding-a-user-application-role)
+  * [Example 4 - Enabling Persistence](#example-4---enabling-persistence)
+* [Cleaning Up](#cleaning-up)  
+
+# Prerequisites
+
+## Coherence Operator Quick Start
+
+Ensure you have followed all the [Quick Start Guide](https://oracle.github.io/coherence-operator/docs-unstable/2.0.0-1910091023/#/about/03_quickstart) including the
```

Comment:
Hard coding links to pre-release docs is probably a bad idea as we will delete them

Reply:
ok thats fine. What will the final link be?